prow/plugins.yaml

```diff
@@ -21,6 +21,7 @@ kubernetes/kubernetes:
 kubernetes/test-infra:
 - trigger
 - config-updater
+- lint
```

Comment:
This is golang-specific as opposed to all the rest of the plugins. Should we prefix with "go"?